src/main/kotlin/uk/gov/gdx/datashare/queue/HmppsQueue.kt

```diff
@@ -0,0 +1,14 @@
+package uk.gov.gdx.datashare.queue
+
+import com.amazonaws.services.sqs.AmazonSQS
+
+class HmppsQueue(
```

Comment:
Have you considered a big ol' rename of everything HMPPS to remove that?

Reply:
I was considering it, but hadn't yet - I can do so now though